config/initializers/omniauth.rb

```diff
@@ -1,17 +1,18 @@
 OmniAuth.config.logger = Rails.logger
 
 Rails.application.config.middleware.use OmniAuth::Builder do
-  provider :facebook, ENV['FACEBOOK_APP_ID'], ENV['FACEBOOK_SECRET'],
-           :scope => 'email', :info_fields => 'name, email'
+  provider :facebook, ENV["FACEBOOK_APP_ID"], ENV["FACEBOOK_SECRET"],
+  :scope => "email", :info_fields => "name, email"
 
   provider :google_oauth2, ENV["GOOGLE_CLIENT_ID"], ENV["GOOGLE_CLIENT_SECRET"]
 
-  provider :twitter, ENV["TWITTER_API_KEY"], ENV["TWITTER_SECRET"], { image_size: 'bigger'}
+  provider :twitter, ENV["TWITTER_API_KEY"], ENV["TWITTER_SECRET"],
+  { image_size: "bigger"}
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Redundant curly braces around a hash parameter.<br>Space inside } missing.